src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -1241,7 +1241,7 @@ private string GetPreferredExtension(DocumentId documentId, SourceCodeKind sourc
 
         public override IVsHierarchy GetHierarchy(ProjectId projectId)
         {
-            _foregroundObject.AssertIsForeground();
+            //_foregroundObject.AssertIsForeground();
```

Comment:
Undoing this change, it was a workaround for a known issue with FSharp to allow testing.